### DISCUSSION TITLE
What is the input for signature matrix calculation?

### DISCUSSION THREAD

**Discussion by RolantusdataExp** [BayraktarLab/cell2location#351] [Q&A]
Dear developers, 
I have a general question: What is the expected input for the single-cell RNA sequencing data used for signature matrix estimation? Is it raw data or normalized data (log-normalized)? 
best,
Peter

**Comment by vitkl** [comment]
It is the integer count data produced by Cellranger or equivalent software - and possibly corrected counts produced by Cellbender but that would need to be validated against the default option. You can read the paper methods section and the tutorial in detail here https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html#Loading-Visium-and-scRNA-seq-reference-data for more detail.
